packages/backend/package.json

```diff
@@ -75,7 +74,6 @@
     "knex": "^0.95",
     "koa": "^2.14.1",
     "koa-bodyparser": "^4.3.0",
-    "lodash": "^4.17.21",
```

Comment:
You'll need to update the pnpm lockfile by running
```
pnpm --filter backend remove lodash
pnpm --filter backend remove @types/lodash
```
Ditto for mock-account-provider